<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QComboBox" name="portListBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="serialInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QComboBox" name="baudRateBox">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="portNameLabel">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="baudRateLabel">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>40</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BaudRate:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="autosampleCheckbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Autosample</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fpsField">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="testSampleButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sample test</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numWorkersSpinBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>fps at</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setColorButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set color</string>
      </property>
     </widget>
     <widget class="QSlider" name="brightnessSlider">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Brightness</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTimeEdit" name="wakeupTimeEditor">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sunriseLengthEditor">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wake up time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sunrise length</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sleep time</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sunsetLengthEditor">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sunset length</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="sleepTimeEditor">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSomething"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSomething">
   <property name="text">
    <string>Something</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
